### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by s0ph1e**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,129 +31,117 @@ var options = {
...
+* [directory](#directory) - path to save files, *required*
+* [sources](#sources) - selects which resources should be downloaded
+* [recursive](#recursive) - follow anchors in html files
+* [maxDepth](#maxdepth) - maximum depth for dependencies
+* [request](#request) - custom options for for [request](https://github.com/request/request)
+* [subdirectories](#subdirectories) - subdirectories for file extensions
+* [defaultFilename](#defaultfilename) - filename for index page
+* [prettifyUrls](#prettifyurls) - prettify urls
+* [ignoreErrors](#ignoreerrors) - whether to ignore errors on resource downloading
+* [urlFilter](#urlfilter)
```
Comment (aivus) [inline README.md]: Some description maybe?

Comment (s0ph1e) [inline README.md]: yes, will add some description

**Comment by s0ph1e** [comment]
I prefer to have documentation in one place instead of having a lot of files. And when everything is in one place it is easier to use `Ctrl+F` to search for something on page. 
It is not overloaded from my point of view.

**Comment by s0ph1e** [comment]
@aivus what do you think in general, is it better than current structure or not?

**Comment by aivus** [comment]
@s0ph1e looks like ok

**Pull request merged**